contributions/demo/jhammarstedt-sestys/README.md

```diff
@@ -0,0 +1,14 @@
+# Demo:  Machine Learning CI/CD with Github Actions and Kubernetes
+
+
+## Members
+* Johan Hammarstedt, johham@kth.se, Github: jhammarstedt
+
+* Matej Sestak, sestak@kth.se, Github: sestys
+
+## Topic
+In this demo we will show how to use Kubernetes and TF serving to develop and deploy a TF/Pytorch model and then automate the full workflow using Github Actions.
+(Alternative idea: if not our original is not accepted we could instead do the same but using Google Cloud)

```

Comment:
Google Cloud is more interesting, please change!

Reply:
Hi @cesarsotovalero, we fixed the folder names and changed the topic a bit to focus specifically on MLops on Google Cloud Platform